src/hooks/useOrgHooks.js

```diff
@@ -195,3 +195,66 @@ export function useTokenBalances(account, token, timer = 3000) {
 
   return balances
 }
+
+export function useCurrencies() {
```

Comment:
Related to the second comment, lets move this into its own file as it's not really an org hook. Consider also declaring the url and Symbol Map outside the hook function.